### PR TITLE
Milestone 2: Added Card grid layout

### PR DESCRIPTION
**Pull request by prg-04**
###I have done the following to reach this milestone:

- [ ] Added the card section using a grid-layout.
- [ ] Made the page to be responsive as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,149 @@
...
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [My_Portfolio] <a name="about-project"></a>
+
```
Comment (shella12) [inline README.md]: - [ ] Great job 👍  using on using the [README template](https://github.com/microverseinc/readme-template) however I would suggest you add a few lines describing your project here in the about section, explaining project's objectives and its essential qualities. Think of it as the elevator pitch to amaze the recruiters 😉

File: index.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+            <li class="tech">HTML</li>
+            <li class="tech">Bootstrap</li>
+            <li class="tech">Ruby</li>
+          </ul>
+          <button class="card_btn">See Project</button>
+        </div>
+      </div>
+    </section>
+
+    <script src="index.js"></script>
```
Comment (shella12) [inline index.html]: - [ ] Great job 👍 adding the work section however you should also include to `about us` section of the project as well as it is mentioned in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements)

> - the "About myself" section (placed after the projects and before the contact form):

**About us section from Figma design Template:**

Screenshot 2023-03-24 at 10 37 13 AM

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,524 @@
...
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+#portfolio .card .tech_stack {
+  display: flex;
+  gap: 1rem;
+  margin: 1rem 0;
+}
```
Comment (shella12) [inline styles.css]: - [ ] Please add a bit of padding in the `title `, `description `and `tech stack` of the project card to match the [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=bFO3LY97dTXnhgBF-0)

Figma design:

Screenshot 2023-03-24 at 10 53 22 AM


Your deisgn:

Screenshot 2023-03-24 at 10 53 10 AM

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,524 @@
...
+  font-size: 40px;
+  line-height: 52px;
+  color: #1c1a19;
+}
+
+#portfolio .title hr {
+  margin-top: 1rem;
+  background-color: #1c1a19;
+  width: 90%;
+}
```
Comment (shella12) [inline styles.css]: - [ ] Please add more margin between the `title ` and the `hr` of the work section as it is the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) to stick to the  [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=bFO3LY97dTXnhgBF-0)
 as much as possible (e.g., font, colors, images, text, margins) 

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Figma design:

Screenshot 2023-03-24 at 10 51 42 AM

Your deisgn:

Screenshot 2023-03-24 at 10 52 07 AM

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,524 @@
...
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+#portfolio .cards .placeholder .plchdr_text ul {
+  display: flex;
```
Comment (shella12) [inline styles.css]: - [ ] Please align `.placeholder` class using flex box as stated in the project requirement 

> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

- [ ] Please add a bit more gap between the elements of the project card to match the [figma design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=bFO3LY97dTXnhgBF-0) as per [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements)

Figma design:

Screenshot 2023-03-24 at 10 51 49 AM

Your deisgn:


Screenshot 2023-03-24 at 10 52 16 AM

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @prg-04 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✅ 
- Professional README.md ✅   
- Correct use of git flow ✅ 
- Correct use of semantic tags ✅ 
- Good commit history ✅ 
- Descriptive pull request ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amon-cofie — APPROVED** [review]
Hi @prg-04 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

🚀 No linter errors
🚀 Descriptive pull request
🚀 Good commit messages 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**